src/js/themes/base.d.ts

```diff
@@ -1617,6 +1617,9 @@ export interface ThemeType {
       };
     };
     value?: TextProps;
+    icons?: {
+      remove?: any;
```

Comment:
Can we adjust this type to be more specific
```suggestion
      remove?: React.ReactNode | Icon;
```